packages/page-accounts/src/modals/Create.tsx

```diff
@@ -262,147 +278,172 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
       header={t<string>('Add an account via seed')}
       size='large'
     >
-      {address && isConfirmationOpen
-        ? (
-          <CreateConfirmation
-            address={address}
-            derivePath={derivePath}
-            isBusy={isBusy}
-            name={name}
-            pairType={pairType}
-            seed={seed}
-          />
-        )
-        : (
-          <Modal.Content>
-            <Modal.Columns>
-              <Modal.Column>
-                <AddressRow
-                  defaultName={name}
-                  noDefaultNameOpacity
-                  value={isSeedValid ? address : ''}
+      <Modal.Content>
```

Comment:
It would be good to have something like "step 1 or 2" in the header.

Reply:
Thought about it, just wasn't sure how well it will play with some of the translations,`ar` in particular. First, it's RTL. Second, some of the translations with numbers do not include number in translation (eg. `"1 day": "يوم واحد"`). So if I did sth like ${t('header') (1/2)}, would it still make sense in arabic?

Reply:
Arabic is currently a problem child. :( It is actually wrong in some places (visibly so). So my view on the header is changing the string to -

`t<string>('add {{step}}/2', { replace: { step } })`

(Will just need to make a sweep to add these to the existing translation files, but shouldn't break - happy to NOT do it is Arabic, I struggle with my tools to even fix the obviously wrong lines)